client/components/experiment/index.tsx

```diff
@@ -42,8 +47,19 @@ export const Experiment: FunctionComponent< ExperimentProps > = ( props ) => {
 	return (
 		<>
 			<QueryExperiments />
-			{ React.Children.map( children, ( elem ) => {
-				return React.cloneElement( elem, { variation, isLoading: loading } );
+			{ React.Children.map( children as JSX.Element[], ( elem ) => {
+				if ( elem ) {
+					const props: LoadingProps = { variation };
+
+					// Unless element is a DOM element
+					if ( 'string' !== typeof elem.type ) {
```

Comment:
React triggers a warning when non standard attributes are added to DOM elements.

Reply:
Weird how variation is allowed? But not issue